davealex/quick-credit
# #166179255 Implement signup API with database

#### What does this PR do?
Test and Implement signup API endpoint with database integration
#### Description of Task to be completed?
- Write tests for `/api/v1/signup` endpoint database query
- Implement database query
#### How should this be manually tested?
Using Postman, make a request  to the API endpoint POST ‘/api/v1/signup’ and ensure that the appropriate JSON responses are returned (when passing validation, and when failing validation)
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166179255](https://www.pivotaltracker.com/story/show/166179255)
#### Screenshots (if appropriate)
#### Questions:
N/A
